### PR TITLE
Add nginx log options

### PR DESCRIPTION
**Pull request by leszekbulawa**
* Allow sending error logs to stdout

* Set log level

#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

Although there already are options to configure logging with persistent volume many solutions, especially for Kubernetes, make use of stdout/stderr when gathering logs.
Also, following the best practices of cloud software development there should be such option without a need of changing nginx config directly.
Empty `kubectl logs <pod>` might increase ones heart rate :)
Ref: http://nginx.org/en/docs/ngx_core_module.html#error_log

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

Some of Nginx docker images are using softlinks to bind log file with stdout/stderr 
`RUN ln -sf /dev/stdout /var/log/nginx/access.log && ln -sf /dev/stderr /var/log/nginx/error.log`

### CODE REVIEW

**Comment by eldada** [comment]
Please rebase and resolve.

File: stable/artifactory-ha/values.yaml [hunk 1]
```diff
@@ -1149,10 +1149,21 @@ nginx:
   #    memory: "128Mi"
   #    cpu: "50m"
 
+  # Logs options
+  logs: {}
+    # Send error logs to stderr insead of a file
+    # stderr: true
```
Comment (danielezer) [inline stable/artifactory-ha/values.yaml]: Please uncomment this part. I think its better to use:
```
logs:
  stderr: false
  level: warn
```

File: stable/artifactory-ha/values.yaml [hunk 2]
```diff
@@ -1149,10 +1149,21 @@ nginx:
...
+    # Set log level - debug, info, notice, warn, error, crit, alert, or emerg
+    # level: warn
+
   mainConf: |
     # Main Nginx configuration file
     worker_processes  4;
-    error_log  {{ .Values.nginx.persistence.mountPath }}/logs//error.log warn;
+    {{ if .Values.nginx.logs.stderr }}
+    error_log  stderr {{ default "warn" .Values.nginx.logs.level }};
```
Comment (danielezer) [inline stable/artifactory-ha/values.yaml]: The suggested change would also make the `default "warn"` part redundant

File: stable/artifactory-ha/values.yaml (hunk 1 above)
Comment (eldada) [reply stable/artifactory-ha/values.yaml]: I agree

File: stable/artifactory/values.yaml [hunk 3]
```diff
@@ -824,17 +824,15 @@ nginx:
...
+    stderr: false
+    level: warn
 
   mainConf: |
     # Main Nginx configuration file
     worker_processes  4;
 
-    {{- if .Values.nginx.logs.stderr }}
+    {{ if .Values.nginx.logs.stderr }}
     error_log  stderr {{ default "warn" .Values.nginx.logs.level }};
```
Comment (danielezer) [inline stable/artifactory/values.yaml]: Please do the same here

**Comment by leszekbulawa** [comment]
Comments addressed

**Comment by danielezer** [comment]
Thanks for the contribution!

**Pull request merged**